firefox-ios/Client/Frontend/TrackingProtection/TrackigProtectionRedux/TrackingProtectionModel.swift

```diff
@@ -126,7 +126,7 @@ class TrackingProtectionModel {
         if !connectionSecure {
             return theme.colors.layerRatingFSubdued
         }
-        return theme.colors.layer3
+        return theme.type == .privateMode ? theme.colors.layerAccentPrivateNonOpaque : theme.colors.layer3
```

Comment:
I believe this goes against the purpose of having a theming system in that we have to check the type of theme instead of setting one color.

Happy to open it to the team, but maybe this is something we need to discuss with the designer?